uber/automated_emails.py

```diff
@@ -115,6 +115,10 @@ def __init__(
         before = [d.active_before for d in when if d.active_before]
         self.active_before = max(before) if before else None
 
+    @property
+    def body(self):
+        return decorators.render_empty(os.path.join('emails', self.template))
```

Comment:
Make `automated_email_fixture.body` a lazy property, so all of the other plugins have a chance to load before rendering.